### PR TITLE
[CEN-1021] Deploy Pipeline

### PR DESCRIPTION
**Pull request by GiovanniMancini**
#### Description

This PR proposes an Azure DevOps pipeline to deploy the batch acquirer in DEV k8s cluster.

#### List of Changes

- A new Deploy Pipeline
- A very basic Docker File, needed by the pipeline
- A basic Deployment manifest for k8s, which should be refined with environment sources

#### Motivation and Context

This PR resolves two problems
- Deploy the application in a controlled environment for testing and quality improvement
- Setup the ground to distribute the app as a docker image

#### How Has This Been Tested?

Live test in Azure DevOps. I attach the k8s log to show that the pod crashes due to lack of a proper environment, which will be fixed in other PRs.

Schermata 2021-11-25 alle 14 27 21

- [ ] Unit Tests
- [ ] Narrow Integration Tests
- [ ] Integration Tests
- [ ] System Tests
- [ ] Performance Tests
- [ ] User Acceptance Tests
- [ ] Smoke Tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Documentation:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      - integration/*
+      - pom.xml
+
+pr: none
+
+resources:
+  - repo: self
+
+variables:
+  # dockerfileRelativePath: '$(DOCKERFILE)' # DockerfileV1
```
Comment (alberto-re-pagopa) [inline .devops/deploy-pipelines.yml:27]: If its not needed as it seems please remove this line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jdk-alpine
+VOLUME /tmp
```
Comment (alberto-re-pagopa) [inline Dockerfile:2]: There is a specific need to specify this volume?

**Review by alberto-re-pagopa — CHANGES_REQUESTED** [review]
Just two minor issues for clarity. Surely other things will change but it's not the case to spend too much time now until we're ready with the parameters needed in order to run properly the JAR.

File: .devops/deploy-pipelines.yml (hunk 1 above)
Comment (GiovanniMancini) [reply .devops/deploy-pipelines.yml:27]: OK, thank you. I'll fix

File: Dockerfile (hunk 2 above)
Comment (GiovanniMancini) [reply Dockerfile:2]: Maybe the application could use it to store temporary files. I don't know, just a guess

Comment (alberto-re-pagopa) [reply Dockerfile:2]: Ok, let's leave that VOLUME in place for now

**Review by alberto-re-pagopa — APPROVED** [review]

**Pull request merged**